docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -301,6 +301,17 @@ Returns :repo:`information <include/request_info/request_info.h>` related to the
 
 Returns a :ref:`request info object <config_http_filters_lua_request_info_wrapper>`.
 
+connection()
+^^^^^^^^^^^^
+
+.. code-block:: lua
+
+  connection = handle:connection()
+
+Returns the current request's underlying :repo:`connection <include/envoy/network/connection.h>`.
+
+Returns a :ref:`metadata object <config_http_filters_lua_connection_wrapper>`.
```

Comment:
"metadata object" ? Is this a copy/paste error?

Reply:
@mattklein123 oops. Sorry about that. Fixed in 22680ac.